### PR TITLE
Add random generator

### LINKED ISSUE #26: Add random number generator
As a tutor who yearns for efficiency due to limited class time, I can do a random number generator to generate a random student to answer my question, instead of having to find online generators

### PR DESCRIPTION
**Pull request by erisjacey**

### CODE REVIEW

File: src/main/java/atas/commons/core/random/RandomGenerator.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        rng = new Random(seed);
+    }
+
+    /**
+     * Returns a new {@code RandomGenerator} with arbitrary seed 1.
+     */
+    public static RandomGenerator makeRandomGenerator() {
+        return new RandomGenerator(SEED);
+    }
+
```
Comment (marcustw) [inline src/main/java/atas/commons/core/random/RandomGenerator.java:28]: Perhaps may want to add in a Javadoc here to make it clearer?

File: src/test/java/atas/logic/commands/sessionlist/AddSessionCommandTest.java [hunk 2]
```diff
@@ -241,6 +241,11 @@ public void setCurrentSessionTrue() {
         public boolean returnCurrentSessionEnabledStatus() {
             return false;
         }
+
+        @Override
+        public Index generateRandomStudentIndex() {
+            return null;
```
Comment (marcustw) [inline src/test/java/atas/logic/commands/sessionlist/AddSessionCommandTest.java]: Just a query, would returning null be a good idea?

File: src/test/java/atas/logic/commands/studentlist/AddStudentCommandTest.java [hunk 3]
```diff
@@ -240,6 +240,12 @@ public void setCurrentSessionTrue() {
         public boolean returnCurrentSessionEnabledStatus() {
             return false;
         }
+
+        @Override
+        public Index generateRandomStudentIndex() {
+            return null;
```
Comment (marcustw) [inline src/test/java/atas/logic/commands/studentlist/AddStudentCommandTest.java]: Just a query, would returning null be a good idea?

**Review by marcustw — COMMENTED** [review]
LGTM! Just some query for the 2 test files.

File: src/test/java/atas/logic/commands/sessionlist/AddSessionCommandTest.java (hunk 2 above)
Comment (erisjacey) [reply src/test/java/atas/logic/commands/sessionlist/AddSessionCommandTest.java]: This implementation is under the ModelStub and this method is not used at all, so I thought returning null wouldn't make much of a difference either way.

Do you have any alternative suggestions for this part?

File: src/test/java/atas/logic/commands/studentlist/AddStudentCommandTest.java (hunk 3 above)
Comment (erisjacey) [reply src/test/java/atas/logic/commands/studentlist/AddStudentCommandTest.java]: Same as above:

This implementation is under the ModelStub and this method is not used at all, so I thought returning null wouldn't make much of a difference either way.

Do you have any alternative suggestions for this part?

File: src/main/java/atas/commons/core/random/RandomGenerator.java (hunk 1 above)
Comment (erisjacey) [reply src/main/java/atas/commons/core/random/RandomGenerator.java:28]: Did I not add it in? Lines 22-24.

Comment (marcustw) [reply src/main/java/atas/commons/core/random/RandomGenerator.java:28]: Sorry, I meant for getNextIndex method!

Comment (marcustw) [reply src/main/java/atas/commons/core/random/RandomGenerator.java:28]: Sorry, I meant for getNextIndex method!

**Review by marcustw — APPROVED** [review]
LGTM!

**Pull request merged**